web_widget_text_markdown/static/src/css/web_widget_text_markdown.css

```diff
@@ -0,0 +1,204 @@
+.o_field_text_markdown {
+    font-size: 15px;
+    line-height: 1.7;
+    overflow: hidden;
+}
```

Comment:
I found the cause for the big amount of whitespace between two paragraphs; it's because Odoo adds

`white-space: pre-wrap`

to class `o_field_text`

In case the `o_field_text` is `o_field_text_markdown` it should just be `white-space: normal`, since the content is not only text but also HTML elements.